campos_potenciais2.py

```
@@ -0,0 +1,278 @@
+    sim.setJointTargetVelocity(l_wheel, 0.0)
+    sim.setJointTargetVelocity(r_wheel, 0.0)
+    time.sleep(0.2)
+    sim.stopSimulation()
+    print("Simulação parada.")
+
+except KeyboardInterrupt:
+    sim.setJointTargetVelocity(l_wheel, 0.0)
+    sim.setJointTargetVelocity(r_wheel, 0.0)
+    sim.stopSimulation()
+    print("Interrompido pelo usuário; sim parada.")
+except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)